### PR TITLE
Change command line flag allow-tools default to false

### PR DESCRIPTION
**Pull request by keertip**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 0.28.4-dev
+* **Breaking change** The `allow-tools` command line flag defaults to false.
```
Comment (devoncarew) [inline CHANGELOG.md]: `Change the default for the '--allow-tools' command-line flag to false.`?

**Comment by keertip** [comment]
Yes, will update the flutter script before publishing a new version. Flutter does pull in only published versions, right?

**Comment by keertip** [comment]
#1940

**Comment by devoncarew** [comment]
> Yes, will update the flutter script before publishing a new version. Flutter does pull in only published versions, right?

Yes, and I believe they use only explicit versions (the version number is referenced in the script).

**Comment by sigurdm** [comment]
Does this also disallow enabling `--allow-tools` from the `analysis_options.yaml` file as discussed in #1940?

**Comment by devoncarew** [comment]
It doesn't, but it should like that is something we'd like to do. I'll open an issue to track; PRs welcome for it as well.

**Comment by keertip** [comment]
The only way to set `--allow-tools` is through command line. It cannot be set thru the dartdoc_options file.

**Comment by sigurdm** [comment]
Ah - that was unclear to me. Thanks!

**Pull request merged**